### ISSUE TITLE
Logo ideas

### ISSUE THREAD

**Issue by auryn-macmillan** [auryn-macmillan/DAOForum#14]
![daohub_sharp](https://cloud.githubusercontent.com/assets/8453294/14043010/5692077e-f2cb-11e5-935c-1e57dbc91abe.png)

**Comment by FelixA** [comment]
![logo_dao](https://cloud.githubusercontent.com/assets/4763120/14065884/64ea3ea6-f43b-11e5-9ed2-6904256160b3.jpg)
![logo_dao_hub](https://cloud.githubusercontent.com/assets/4763120/14065885/6c148b1e-f43b-11e5-87d5-bc1ef8b0d7d5.jpg)

**Comment by auryn-macmillan** [comment]
@FelixA I like it, simple. Are you suggesting the first one might be the symbol for the DAO? What made you choose that specific green?

**Comment by vmichalik** [comment]
![city-of-melbourne-logo-variations](https://cloud.githubusercontent.com/assets/12459761/14113390/aad08246-f5ca-11e5-9c50-54d5e5d0fc93.jpg)
2015-16 logo trend seems to be flat colour/polygon. I'll try mock something up in illustrator to show what I mean. I really like the concept of the Tao hub thing though, maybe we can play on that theme but a little more abstract.

**Comment by auryn-macmillan** [comment]
agreed, that definitely seems to be the current trend. Looking forward to your suggestions.

**Comment by matrixator** [comment]
![dao_logo_1 4](https://cloud.githubusercontent.com/assets/18177240/14158340/416a8892-f6d8-11e5-9003-5ead1a15cc84.jpg)

**Comment by patrickhidalgo** [comment]
Very clean.  I really like how the arrows are a play on decentralization.  I still haven't made up my mind on weather DAO should be the name because it is the thing, not the brand (like Car instead of Ford).

**Comment by auryn-macmillan** [comment]
@matrixator @patrickhidalgo: here we're looking for ideas for a logo for the daohub forum, rather than the DAO its self.

**Comment by matrixator** [comment]
OK. I like your version. It looks like a parliament :-).
